tests/Unit/Utilities/PregTest.php

```diff
@@ -346,13 +338,12 @@ public function splitReturnsArrayContainingSubjectOnError(): void
     /**
      * @test
      */
-    public function splitWithOffsetCaptureReturnsArrayContainingArrayWithSubjectAndZeroOffsetOnError(): void
+    public function splitWithOffsetCaptureIsNotSupported(): void
     {
+        $this->expectException(\RuntimeException::class);
```

Comment:
Please let's also test for the exception message and the code.

Reply:
Done.